### ISSUE TITLE
508-defect-2 [SCREENREADER]: Adding disability, year of decision, NVDA does not announce year or selection status

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#25176]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect form elements to announce their status when changed.
<h2>Details</h2>
When adding a disability, there is a spinner which allows the user to either select or type the year in which a decision was rendered regarding that disability. When I type a year, NVDA does not announce the year or its selection status. This bug does not occur when using JAWS.
<h2>Acceptance Criteria</h2>
The year and selection status are announced in all environments.
<h2>Environment</h2>
Chrome with NVDA
<h2>Steps to Recreate</h2>
<ul>
<li> Access

https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/introduction

And sign in with vetsgov+test238@id.me </li>
<li> Proceed to step two and add a disability</li>
<li>Using NVDA, type "2021" in the "year" field.</li>
</ul>

**Comment by Mottie** [comment]
This is an issue with the global date widget. cc @caw310

**Comment by caw310** [comment]
According to Andrew, this is an issue with the REACT date component which has been deprecated. Is this still an issue with va-date web component?  How can that widget get updated with the web component?

**Comment by rsmithadhoc** [comment]
As this was part of React's date component and we have officially moved to v3 with web components, which does not have the same problem, I will close this issue.
